### PR TITLE
add new exporter

### PR DESCRIPTION
**Pull request by thopp-tudelft**
add new exporter code, directly exports from render api to webknossos

adds formatting script and adds arg to basic auth script as well

### CODE REVIEW

**Comment by arentkievits** [comment]
Can we fetch `voxel_size` from render? It is included in the stack metadata. Currently you can only set a single voxel size, but if you have multiple stacks with different voxel sizes (e.g. for CLEM data) this is problematic.

File: scripted_render_pipeline/importer/__main__.py [hunk 1]
```diff
@@ -12,36 +13,40 @@
...
     (NAS_SHARE_PATH if REMOTE else pathlib.Path(SERVER_STORAGE_PATH_STR))
     # / "thopp/20191101_rat-pancreas_partial"
     #  / "rlane/SECOM/projects/20191101_rat-pancreas_partial"
-    / "thopp/20230523_singleholder_Earthworm_03/ROA-1"
+    / f"akievits/FAST-EM/tests/{PROJECT}"
 )
 MIPMAP_TYPE = "FASTEM"  # "CLEM"
 # for fastem datasets only
 USE_POSITIONS = True  # use the automated stitching results
+MULTIPLE_SECTIONS = True # Set to False for a single section
```
Comment (thopp-tudelft) [inline scripted_render_pipeline/importer/__main__.py:36]: setting this to false would raise a NameError at line 49

File: scripted_render_pipeline/exporter/webknossos_exporter.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        set to 0 for no downsampling
+    processes: how many parallel processes to use for downscaling
+        defaults to 8
+    """
+
+    def __init__(
+        self,
+        location,
+        *args,
+        downscaling,
```
Comment (thopp-tudelft) [inline scripted_render_pipeline/exporter/webknossos_exporter.py:41]: downscaling is an arg of the parent class, redefining it and then not providing it to the base class causes it to always be the default value

File: scripted_render_pipeline/importer/__main__.py (hunk 1 above)
Comment (arentkievits) [reply scripted_render_pipeline/importer/__main__.py:36]: Allright, then set PROJECT_PATHS either to the natsorted list or to "None"?

**Pull request merged**